docs/docs/examples/structured_outputs/structured_outputs.ipynb

```diff
@@ -67,7 +67,7 @@
    "outputs": [],
    "source": [
     "from typing import List\n",
-    "from pydantic.v1 import BaseModel, Field\n",
+    "from pydantic import BaseModel, Field\n",
```

Comment:
i think maybe we should import from our bridge instead?

```python
from llama_index.core.bridge.pydantic import BaseModel, Field
```

Reply:
this is a notebook though showing how users can integrate llamaindex into the rest of their code, not our own internal code. are we trying to say that if users use llamaindex, they should always use `llama_index.core.bridge.pydantic` and now default `pydantic` imports?

Reply:
Yeah that's a fair point. I think the point I guess is the same as with any attempting to integrate with any other framework... if there is a common dependency in this case pydantic, then I think its reasonable to assume that there is a chance things can go wrong if the common dependency is not the same version.

Using the bridge would just be a safe way to ensure that you're using the same pydantic dep in the users code. If they don't then yeah they just need to use the correct version manually.

Reply:
I think since now we pin `pyndatic>=2.0,<3.0` in the deps for core, this shouldn't be an issue

Reply:
Importing from bridge before was nice because we supported both v1 and v2

Reply:
ah okay, my thinking on this is that it was nice to sort of guarantee some level of safety/compatibility when using Pydantic.

but i guess we don't do this for any other deps so yeah